src/BizHawk.Client.Common/RomLoader.cs

```diff
@@ -978,6 +978,8 @@ private static class RomFileExtensions
 
 			public static readonly IReadOnlyCollection<string> A78 = new[] { "a78" };
 
+			public static readonly IReadOnlyCollection<string> _3DO = new[] { "3do-iso" };

```

Comment:
Absolutely not. 3DO is a CD console, and looks to just be accepting normal CD files. Users should not be renaming their CD files to "3do-iso", they should be loading them up as normal and that goes through the standard BizHawk CD interface, which this core MUST use to be considered acceptable for merge.

Reply:
Discussed this on Discord, this was apparently used more for testing purposes before the CD interface was actually hooked up (done last).

Still, doing this is a dealbreaker, and makes this unacceptable for merge until that is done at least.

Reply:
fixed